src/software/thunderscope/play/refereeinfo_widget.py

```diff
@@ -28,72 +29,44 @@ def __init__(self, buffer_size=5):
         QWidget.__init__(self)
 
         self.referee_table = QTableWidget(
-            refereeInfoWidget.NUM_ROWS, refereeInfoWidget.NUM_COLS
+            RefereeInfoWidget.NUM_ROWS, RefereeInfoWidget.NUM_COLS
         )
         self.referee_info = QLabel()
         self.referee_buffer = ThreadSafeBuffer(buffer_size, Referee, False)
+        self.referee_table.verticalHeader().setVisible(False)
 
         self.vertical_layout = QVBoxLayout()
         self.vertical_layout.addWidget(self.referee_table)
         self.vertical_layout.addWidget(self.referee_info)
         self.setLayout(self.vertical_layout)
 
-    def set_data(self, data):
-        """Data to set in the table
-
-        :param data: dict containing {"column_name": [column_items]}
-
-        """
-        horizontal_headers = []
-
-        # empirically makes even bolded items fit within columns
-        HEADER_SIZE_HINT_WIDTH_EXPANSION = 12
-        ITEM_SIZE_HINT_WIDTH_EXPANSION = 11
-
-        for n, key in enumerate(data.keys()):
-            horizontal_headers.append(key)
-
-            for m, item in enumerate(data[key]):
-                str_item = str(item)
-                newitem = QTableWidgetItem(str_item)
-                newitem.setSizeHint(
-                    QtCore.QSize(
-                        max(
-                            len(key) * HEADER_SIZE_HINT_WIDTH_EXPANSION,
-                            len(str_item) * ITEM_SIZE_HINT_WIDTH_EXPANSION,
-                        ),
-                        1,
-                    )
-                )
-                self.referee_table.setItem(m, n, newitem)
-
-        self.referee_table.setHorizontalHeaderLabels(horizontal_headers)
-
     def refresh(self):
-        """Update the play info widget with new play information
+        """Update the referee info widget with new referee information
         """
         referee = self.referee_buffer.get(block=False)
         referee_msg_dict = MessageToDict(referee)
 
-        if referee_msg_dict:
-            p = (
-                f"Packet Timestamp: {round(float(referee_msg_dict['packetTimestamp'])*SECONDS_PER_MICROSECOND, 3)}\n"
-                + f"Stage: {referee_msg_dict['stageTimeLeft']*SECONDS_PER_MICROSECOND} {referee_msg_dict['stage']}\n"
-                + "Command: "
-                + referee_msg_dict["command"]
-                + "\n"
-                + f"Blue Team on Positive Half: {referee_msg_dict['blueTeamOnPositiveHalf']}\n"
-            )
-            self.referee_info.setText(p)
+        if not referee_msg_dict:
+            return
+
+        SECONDS_PER_MINUTE = 60
```

Comment:
Just to stay consistent, can you add this to `src/shared/constants.h` and using Pybind (`src/software/py_constants.cpp`) make it accessible to Python?